src/services/scoring/methods/matrix-score.test.js

```diff
@@ -0,0 +1,81 @@
+import { ScoreBands } from '../../../config/score-bands.js'
+import multiScore from './multi-score.js'
+import matrixScore from '~/src/services/scoring/methods/matrix-score.js' // adjust path as needed
+
+/**
+ * Question config.
+ * @type {import("../../../config/scoring-types.js").Question}
+ */
+const questionConfig = [
+  {
+    id: 'matrixScore',
+    scoreDependency: 'matrixDependency',
+    answers: [
+      { answer: 'A', score: { A: 1, B: 2, C: 3, D: 4 } },
+      { answer: 'B', score: { A: 2, B: 4, C: 6, D: 8 } }
+    ],
```

Comment:
I understand that the existence of C and D isn't necessary to test other answer's dependency on them, but I think it makes the tests unclear.